### PR TITLE
initialize create react app

### PR DESCRIPTION
**Pull request by angelafrancisco**
initialized create react app, chose react because I have the most experience using react

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (bayounetta) [inline public/robots.txt]: Can we remove this file?

**Review by bayounetta — CHANGES_REQUESTED** [review]
Looks great just had one requested change.

File: public/robots.txt (hunk 1 above)
Comment (angelafrancisco) [reply public/robots.txt]: Yes, removed!

**Review by bayounetta — APPROVED** [review]
Looks Awesome! Thank you so much!

**Pull request merged**